frontend/Gruntfile.js

```diff
@@ -25,6 +21,17 @@ module.exports = function(grunt) {
         }
       }
     },
+    babel: {
+        options: {
+            sourceMap: true,
+            presets: ['es2015']
+        },
+        dist: {
+            files: {
+              '../pimpmycause/static/scripts/pimpmycause.min.js':'src/scripts/scripts.js',
```

Comment:
The strings `../pimpmycause/static` and `src` are repeated many times in the file, consider adding on the top of the file somenthing like:
```var SOURCE_DIR = '../pimpmycause/static';
var DESTINATION_DIR = 'src';```

This will made the code more readable and this gruntfile reusable.